cicd/internal/flags/it-flags.go

```diff
@@ -82,6 +84,13 @@ func PrivateConnectivity() string {
 	return ""
 }
 
+func SpannerHost() string {
+	if dSpannerHost == "" {
```

Comment:
Formatting looks funky here - could you run `go fmt` on the files here? Probably why the check is failing